app/views/artists/index.html.erb

```diff
@@ -1,115 +1,134 @@
       <% @artists.each do |artist| %>
-      <li> <%= link_to artist.name, artist_path(@artist, :id => artist.id) %> &nbsp;&nbsp;
-        <%= link_to "x", artist, method: :delete, data: { confirm: 'Are you sure?' } %></li>
+        <li> <%= link_to artist.name, artist_path(@artist, :id => artist.id) %> &nbsp;&nbsp;
+        <%= link_to "x", artist, method: :delete, data: { confirm: 'Are you sure?' }, id: "remove" %></li>
       <% end %>
     </u>
   </div>
 </div>
 <div class="parallax-page">
-<div class="parallax-container">
-  <div class="parallax">
-    <% if @images.blank? %>
-      <%= image_tag ("Stock-Concert-Photo.jpg") %>
-    <% else %>
-        <% @images.each do |a| %>
-        <%= image_tag a.css('.artist-content img')[1].attr('src') %>
+  <div class="parallax-container">
+    <div class="parallax">
+      <% if @images.blank? %>
+        <%= image_tag ("Stock-Concert-Photo.jpg") %>
+      <% else %>
+          <% @images.each do |a| %>
+          <%= image_tag a.css('.artist-content img')[1].attr('src') %>
+          <% end %>
+      <% end %>
+    </div>
+  </div>
+  <div class="section">
+    <div class="row container">
+      <div class="biography">
+        <h2> BIOGRAPHY </h2>
+        <% if @wikiscrape.blank? %>
+          <p> Couldn't find biography of <%= @artist.name %> </p>
+        <% else %>
+        <% @wikiscrape.each do |a| %>
+          <p><%= a.css('p')[0].text unless a.css('p')[0].text.length < 100  %></p>
         <% end %>
-    <% end %>
+      <% end %>
+    </div>
+    </div>
   </div>
-</div>
-<div class="section">
-  <div class="row container">
-    <div class="biography">
-      <h2> Biography </h2>
-      <% if @wikiscrape.blank? %>
-        <h3> Couldn't find biography of <%= @artist.name %> </h3>
+  <div class="parallax-container">
+    <div class="parallax">
+      <% if @images.blank? %>
+        <%= image_tag ("tc_and_topcats.jpg") %>
       <% else %>
-      <% @wikiscrape.each do |a| %>
-        <p><%= a.css('p')[0].text unless a.css('p')[0].text.length < 100  %></p>
+        <% @images.each do |a| %>
+        <%= image_tag a.css('.artist-content img')[2].attr('src') %>
+        <% end %>
       <% end %>
-    <% end %>
+    </div>
   </div>
+  <div class="section black">
+    <div class="row container">
+      <div class="news">
+      <h2>NEWS</h2>
+          <% if @mtvnews.blank? %>
+            <p> Couldn't find news of <%= @artist.name %> </p>
+          <% else %>
+            <% @mtvnews.each do |a| %>
+            <p><%= raw a.css('.multiline')[0] %></p>
+            <p><%= raw a.css('.multiline')[4] %></p>
+            <p><%= raw a.css('.multiline')[8] %></p>
+            <p><%= raw a.css('.multiline')[12] %></p>
+            <p><%= raw a.css('.multiline')[16] %></p>
+            <p><%= raw a.css('.multiline')[20] %></p>
+            <p><%= raw a.css('.multiline')[24] %></p>
+            <p><%= raw a.css('.multiline')[28] %></p>
+            <p><%= raw a.css('.multiline')[32] %></p>
```

Comment:
Als we deze sequence volgen, slaan we steeds een news item over. Ik pas dit wel aan in een aparte commit